export const updateAvatar = async(req, res) => {
  try {
    
  } catch (error) {
    console.log("Error in updating avatar", error.message);
  }
}
export const updateUser = async (req, res) => {
  try {
    
  } catch (error) {
    console.log("Error in updating user", error.message);
  }
}

export const addAddress = async (req, res) => {
  try {
    
  } catch (error) {
    
  }
}
 
export const updateAddress = async (req, res) => {
  try {
    
  } catch (error) {
    
  }
}